capstone/src/main/java/com/google/univiz/api/dataviz/Deadline.java

```diff
@@ -0,0 +1,3 @@
+package com.google.univiz.api.dataviz;
+
```

Comment:
add JavaDoc event to event empty objects

Reply:
Ah good catch, I overlooked that. Will do.